dd-sdk-android/src/main/kotlin/com/datadog/android/rum/tracking/ActivityViewTrackingStrategy.kt

```diff
@@ -94,6 +94,28 @@ class ActivityViewTrackingStrategy @JvmOverloads constructor(
 
     // endregion
 
+    // region Object
+
+    override fun equals(other: Any?): Boolean {
+        if (this === other) return true
+        if (javaClass != other?.javaClass) return false
+
+        other as ActivityViewTrackingStrategy
+
+        if (trackExtras != other.trackExtras) return false
+        if (componentPredicate != other.componentPredicate) return false
```

Comment:
here you want to compare by value or by reference ? I think it should be by reference (!==)

Reply:
No I want to compare by value